src/CarroMessenger.php

```diff
@@ -27,6 +30,23 @@ public function sendMessage($data)
             );
     }
 
+    /**
+     * Send whatsapp template message with Message Bird
+     *
+     * @param array $data
+     * 
+     * @return void
+     */
+    public function sendWhatsAppTemplateSMSViaMsgBird($data)
+    {
+        $response = WhatsAppMessageBird::sendTemplateMessage($data);
+        $model = data_get($data, 'model');
+
+        if (config('carromessenger.event_is_called') && !is_null($model)) {
```

Comment:
```suggestion
        if (config('carro_messenger.event_is_called') && !is_null($model)) {
```
Why not use underscore? bro

Reply:
updated at commit 859554632e1ce34571085a6ce82a9e5d39ab5ec9